server/routes/api/documents/documents.ts

```diff
@@ -1701,4 +1701,48 @@ router.post(
   }
 );
 
+router.post(
+  "documents.empty_trash",
+  auth({ admin: true }),
```

Comment:
Just changed this to

```suggestion
  auth({ role: UserRole.Admin }),
```